coeus-impl/src/main/resources/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetSettings.xml

```diff
@@ -30,7 +30,7 @@
 				<property name="items">
 					<list>
 						<bean parent="Uif-PrimaryActionButton" p:actionLabel="Apply Changes" p:methodToCall="saveBudgetSettings"
-							p:order="10" p:refreshId = "PropBudget-BudgetSettings-Dialog" p:dialogDismissOption=""
+							p:order="10" p:dialogDismissOption="REQUEST"
```

Comment:
I'm not sure. Maybe @bsmith83 can suggest something. Somehow we need to detect errors and not close the dialog in that case, if it succeeds we'll need to close the dialog and refresh the page. Maybe a callback when the dialog is destroyed that causes a refresh?